core/src/main/scala/Manifest.scala

```diff
@@ -387,7 +387,9 @@ object Manifest {
    * place i could find to put it.
    */
   private def parseDeployable(release: Github.Release, name: String): Task[Deployable] = {
-    release.findAssetContent(s"${name}.deployable.yml").flatMap { a =>
+    release.findAssetContent(s"${name}.deployable.yml").handleWith {
+      case ProblematicDeployable(_, _) => release.findAssetContent(s"${name}.deployable.yaml")
+    }.flatMap { a =>
```

Comment:
This is going to make the error message reference `.yaml` instead of `.yml`, which I guess is not a big deal since both are now supported.  I thought about suggesting `findAssetContent` should take a predicate, but that complicates the error message generation.

I think I'm fine with this.  Just thinking out loud.